samples/fabric/fabric-cli/scripts/getTokenBalance.sh

```diff
@@ -0,0 +1,7 @@
+cd scripts
+
+USER_CERT_BASE64=$(node getAssetTransferViewAddress.js getusercert $1 $2)
+
+cd ..
+
+./bin/fabric-cli chaincode query mychannel simpleassettransfer GetBalance '["token1","'$USER_CERT_BASE64'"]' --local-network=$1
```

Comment:
Is there a reason to do this way? In other `simpleasset`, we assume caller is the one who is querying his own balance (not allowing others to query balance of others).
And to set the caller user, in `query`/`invoke` you need pass extra argument `--user=<user-name>`.